### DISCUSSION TITLE
Appending path to short url

### DISCUSSION THREAD

**Discussion by johnjameswhitman** [thedevs-network/kutt#445] [Q&A]
I have a couple of URL's I'd like to shorten from:
- `https://some-long-domain-name.com/foo/bar` &rarr; `https://my-kutt.com/foo/bar`, and
- `https://some-long-domain-name.com/foo/baz` &rarr; `https://my-kutt.com/foo/baz`

Is it possible to craft a single shortened URL that allows you to append a sub-path to a prefix, e.g. `https://my-long-blog-domain-name.com/foo/{subpath}` &rarr; `https://my-kutt.com/foo/{subpath}`? If I create a shortened URL for `https://my-long-blog-domain-name.com/foo/` &rarr; `https://my-kutt.com/foo`, then I get a 404 when visiting `https://my-kutt.com/foo/bar`.

**Comment by poeti8** [comment]
This is not available right now. Feel free to create an issue so we can investigate.
